homeassistant/components/tibber/sensor.py

```diff
@@ -71,6 +69,25 @@ def state(self):
         """Return the state of the device."""
         return self._state
 
+    @property
+    def device_id(self):
```

Comment:
Shouldn't you also define a unique ID for this sensor using device_id?

Reply:
A unique id is defined here https://github.com/home-assistant/core/pull/34469/files#diff-394a88dffeddcf8d41445b937eedb628L126
Or should it be opposite, so that the unique id is defined from the device id?

Reply:
Oh I see now, ok.